### ISSUE TITLE
Home World Blues

### ISSUE THREAD

**Issue by Spoonikle** [StargateMC/Server#38]
Home Worlds will be heavily restricted to allow ONLY their OWNER access.

Gate Travel to home worlds by any player who is not the owner of target home world will be blocked on 2 fronts - dialing failure and in the event of "tag along" complete bypass of the gates teleport event.

Failed attempts to enter another players home world will notify players about the limits of home worlds and give them info on establishing a faction on public worlds and playing with friends

Players will be told that only they can access their home world by thor when transported to further notify players in advance of friends joining the game.

Home worlds are to be loaded from a seed to provide new players with a Stargate, pre-built and fully upgraded - as well as shelter for their first nights in game. Player kits will no longer contain a stargate as public gates will provide new players with access if they set up a faction on public worlds to play with friends.

These Massive changes are to refocus the group play outside of home worlds, those who wish to play together must occupy and control land in one of the 25 public "overworlds" home worlds are for starting resources / solo play / security.

If such changes do not bring about desired goal, they need to be re-evaluated

**Comment by DraksterAU** [comment]
Moving most of the other discussions to another issue.
